Github/sarif-output.yml

```diff
@@ -0,0 +1,37 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Sarif
+
+# Controls when the workflow will run
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+    branches:
+      - master
+      - main
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # This step checks out a copy of your repository.
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Checkmarx scan
+        uses: checkmarx/ast-github-action@2.0.7
+        with:
+          base_uri: https://ast-master-components.dev.cxast.net/
```

Comment:
Can you use either an environmental variable or a prod instance (i.e. https://ast.checkmarx.net)

Reply:
Updated the reference to use the prod instance